a10_neutron_lbaas/a10_config.py

```diff
@@ -90,10 +92,42 @@ def __init__(self):
                         if dk not in self.devices[k]:
                             self.devices[k][dk] = dv
 
+                    LOG.debug("A10Config, device %s=%s", k, self.devices[k])
+
+            # Setup db foo
+            if self.config.use_database and self.config.database_connection is None:
+                self.config.database_connection = self._get_db_string()
+
         finally:
             sys.path = real_sys_path
 
-        LOG.debug("A10Config, devices=%s", self.devices)
+    # We don't use oslo.config here, in a weak attempt to avoid pulling in all
+    # the many openstack dependencies. If this proves problematic, we should
+    # shoot this manual parser in the head and just use the global config
+    # object.
+    def _get_db_string(self):
+        z = 'sqlite:////tmp/a10.db'
+
+        neutron_conf = '/etc/neutron/neutron.conf'
```

Comment:
What's the location of the neutron.conf file in an HP helion virtual environment? We should be getting this from the operating system environment.